### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by apiraino**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+### Repository for playing with the triagebot
```
Comment (apiraino) [inline README.md:1]: sounds like some content is missing in this README

Comment (apiraino) [reply README.md:1]: @rustbot author

Comment (apiraino) [reply README.md:1]: @rustbot ready

Comment (apiraino) [reply README.md:1]: @rustbot author

Comment (tglman) [reply README.md:1]: @rustbot ready

Comment (apiraino) [reply README.md:1]: @rustbot author

Comment (apiraino) [reply README.md:1]: @rustbot author

Comment (tglman) [reply README.md:1]: @rustbot ready

Comment (apiraino) [reply README.md:1]: well well well

@rustbot ready

Comment (apiraino) [reply README.md:1]: > well well well well
> 
> @rustbot ready

@rustbot ready

Comment (jman-schief) [inline README.md:1]: test

@rustbot ready

**Review by jman-schief — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jman-schief) [reply README.md:1]: r? @apiraino

**Review by jman-schief — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jman-schief) [inline README.md:1]: test

@rustbot ready

**Review by jman-schief — COMMENTED** [review]
this is a comment

**Pull request closed without merging**